library/dd-sdk-android-trace/build.gradle.kts

```diff
@@ -43,6 +43,8 @@ android {
         minSdk = AndroidConfig.MIN_SDK
         targetSdk = AndroidConfig.TARGET_SDK
         setLibraryVersion()
+
+        consumerProguardFiles("consumer-rules.pro")
```

Comment:
what is the difference from :
`
proguardFiles(...)
`
Is this the new way of adding a file or this is just merging with the default project file ?

Reply:
This makes that file to be shipped with `AAR`, so it is consumed later on the app side and rules are taken into the account. We already doing the similar for main module.